arch/arm64/boot/dts/qcom/sdm450-motorola-ali.dts

```diff
@@ -0,0 +1,258 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
@julianbraha Please confirm the sign off for the commit with the name and email from the commit (Julian Braha), then I'll add it to the commit message

https://www.kernel.org/doc/html/latest/process/submitting-patches.html#sign-your-work-the-developer-s-certificate-of-origin

Reply:
Signed-off-by: Julian Braha <julianbraha@gmail.com>